spec/computer_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require_relative '../lib/computer'
+require_relative '../lib/grid'
+require_relative '../lib/grid_display'
+require_relative '../lib/ships_list'
+require_relative '../lib/create_ship'
+require_relative '../lib/ship'
+
+RSpec.describe Computer do
+
+  let(:grid) {Grid.new(10)}
+  let(:create_ship) {CreateShip.new}
+  let(:ships_list) {ShipsList.new(create_ship)}
+  let(:grid_display) {GridDisplay.new(grid.size)}
+  let(:ship) {Ship.new("submarine", 1)}
+  let(:computer) {Computer.new(grid, ships_list, ship)}
+
+
+  xit "places ship on grid from list of 6 ships randomly" do
```

Comment:
I kind of ignored computer in this branch as it would have been dealt with in the next branch. Although I actually made some changes on computer in this branch too. I think I just forgot to "dis-comment" it out (is this the technical word?)